nautobot_golden_config/nornir_plays/config_intended.py

```diff
@@ -111,6 +111,7 @@ def config_intended(nautobot_job, data):
     logger = NornirLogger(__name__, nautobot_job, data.get("debug"))
 
     qs = get_job_filter(data)
+    logger.log_debug("Mapping devices to config context data. This may take a while with a large number of devices.")
```

Comment:
`Mapping devices to config context data` would not resonate to me, additionally, I don't think we need to mention that the process may take a long time, as I think it is understood that running against a lot of devices will take time.